### DISCUSSION TITLE
Specific provider usecase

### DISCUSSION THREAD

**Discussion by Glutch** [nextauthjs/next-auth#1613] [Help] [answered]
I'm building a "gaming community" website, closely connected to a discord community. So i need the users to sign up with Discord. Then optionally also add Steam so i can grab their game data.

I would like to only show the discord signin initially, and somewhere else on the site have a button that says "connect steam". Is this possible with nextauth?

**Comment by Glutch** [comment, marked as answer]
Apparently you just pass signIn('discord'). Wow, that's unbelievably simple

**Reply by ilyafish** [reply]
Is there a code example in the documentation or one you have on how you made this? I'm trying to do the same thing with a SAAS app (sign up with Google, then connect other services)

**Reply by balazsorban44** [reply]
https://next-auth.js.org/getting-started/client#starts-google-oauth-sign-in-flow-when-clicked

You just pass the `id` of your provider to `signIn`
